owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewVideoActivity.kt

```diff
@@ -80,8 +83,10 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
 import org.koin.androidx.viewmodel.ext.android.viewModel
+import org.koin.core.parameter.parametersOf
 import timber.log.Timber
 
+@RequiresApi(Build.VERSION_CODES.TIRAMISU)
```

Comment:
Hmmm I don't like this, this way you're restricting this class to SDK 33+ (Android 13+) devices. Why is that needed?

Reply:
This is necessary if `getParcelableExtra` method has a second parameter, but I've changed it and that annotation has been removed. ✅